### PR TITLE
update voodoo to fix descent

### PR DESCRIPTION
**Pull request by NikkyAI**
sorry for the large amount of diff in lockfiles

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 plugins {
     wrapper
-    id("voodoo") version "0.5.0-SNAPSHOT"
+    id("voodoo") version "0.5.20-SNAPSHOT"
```
Comment (skyem123) [inline build.gradle.kts:3]: Just to confirm, I should use the refreshVersions thing to see when to update this?

File: descentfrozenhell/mods/thermal-foundation.lock.json [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-                "cofh-world"
-            ]
+            "cofh-core": "REQUIRED",
+            "cofh-world": "REQUIRED"
         }
     },
     "projectID": 222880,
-    "fileID": 2926428
+    "fileID": 2926428,
+    "skipFingerprintCheck": true
```
Comment (skyem123) [inline descentfrozenhell/mods/thermal-foundation.lock.json:11]: Why is this true for every curse thing now?

**Review by skyem123 — APPROVED** [review]
It looks good, I just have a few things I want to understand for the future.

File: build.gradle.kts (hunk 1 above)
Comment (NikkyAI) [reply build.gradle.kts:3]: no.. i meant i should show people how to use refreshVersions .. that will be part of the readme and 0.6.0 upgrade... until then no 0.5.X stuff unless something really breaks horribly

Comment (skyem123) [reply build.gradle.kts:3]: I see. Makes sense.

**Pull request merged**